//=================================================================================================
// Copyright (C) 2023-2024 HEPHAESTUS Contributors
//=================================================================================================

syntax = "proto3";

package heph.types.proto;

message NumericValue {
  oneof value {
    int32 int32_value = 1;
    int64 int64_value = 2;
    uint32 uint32_value = 3;
    uint64 uint64_value = 4;
    float float_value = 5;
    double double_value = 6;
  }
}
